src/apdu_pubkey.c

```diff
@@ -98,17 +120,22 @@ size_t handle_apdu_get_public_key(uint8_t _U_ instruction) {
 
     read_bip32_path(&G.key, dataBuffer, cdata_size);
 
-    generate_public_key(&G.public_key, &G.key);
+    generate_public_key(&G.ext_public_key, &G.key);
 
     // write tags
     GPriv.prefixed_public_key_hash.address_type_is_short = 0x01;
     GPriv.prefixed_public_key_hash.key_hash_type_is_sighash = 0x00;
 
     // write lock arg
-    generate_lock_arg_for_pubkey(&G.public_key, &GPriv.prefixed_public_key_hash.hash);
+    generate_lock_arg_for_pubkey(&G.ext_public_key.public_key, &GPriv.prefixed_public_key_hash.hash);
 
     // instruction == INS_PROMPT_PUBLIC_KEY || instruction == INS_AUTHORIZE_BAKING
```

Comment:
This is also a stale Tezos comment, please remove these if close to where you are working